blant.c

```diff
@@ -1100,6 +1100,12 @@ void LoadMagicTable()
 	}
 }
 
+void SetBlantDir() {
+	char* temp = getenv("BLANT_DIR");
+	if (temp != NULL)
+		_BLANT_DIR = temp;
```

Comment:
Only noticed this after I accepted the pull request: on some systems, the space storing the string returned by getenv is re-used on later calls, so you have to copy it. I've already added the appropriate fix in a later commit, by calling strdup().